### ISSUE TITLE
account for students who dropped out of MESM program in career data

### ISSUE THREAD

**Issue by samanthacsik** [UCSB-MEDS/shiny-dashboard#17]
After conversation with KB & MS on 3/28: total graduating class size (for MESM) does not account for students who dropped out/deferred -- this is especially apparent for the incoming class of 2022 (graduating class of 2022), which saw a large drop in enrolled students due to COVID (dashboard currently says the graduating class size was 92, but really in the 70s).

SC waiting on additional data to do this,
